cpp/openlocationcode.cc

```diff
@@ -55,11 +54,15 @@ double compute_precision_for_length(int code_length) {
   return pow_neg(internal::kEncodingBase, -3) / pow(5, code_length - 10);
 }
 
-// Finds the position of a char in the encoding alphabet.
+// Returns the position of a char in the encoding alphabet, or -1 if invalid.
 int get_alphabet_position(char c) {
```

Comment:
Could you add a unit test for this? Something that goes through each character in the alphabet and ensures the position is as expected.

Reply:
The best I could do (which I've now done) is add a unit test to verify consistency between kPositionLUT and kAlphabet. It's not possible to check get_alphabet_position() directly (I think?) because it's in an anonymous namespace and therefore inaccessible from unit test code. Maybe it should be moved into "internal" namespace?

Reply:
I think what you have here is fine. I'm more concerned about the alphabet and look up table being out of sync.